openvino/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# ******************************************************************************
+# Copyright 2017-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+if(NOT COMMON_LIBS_INCLUDED)
+    set(COMMON_LIBS_INCLUDED TRUE PARENT_SCOPE)
```

Comment:
why do we need such guard? Looks like if we add subdirectory `openvino/itt` only once, this is not needed

Reply:
During style check step in CI NGraph is building separately. I included common libs directory to ngraph too. Therefore this guard is nessesary.

Reply:
See standard way https://cmake.org/cmake/help/v3.13/command/include_guard.html it can be included into `opevino/itt/CMakeLists.txt`

Reply:
BTW, @azhogov can we fix ngraph code style check somehow (e.g. use root CMakeLists.txt) not to include WA in our code?

Reply:
> See standard way https://cmake.org/cmake/help/v3.13/command/include_guard.html it can be included into `opevino/itt/CMakeLists.txt`

Ok, thanks. (:

Reply:
I checked documentation about include_guard.
Unfortunatelly it is supported only since CMake 3.10.
I found include_guard usage in inference-engine/cmake/vpu_dependencies.cmake. 

```
if(CMAKE_VERSION VERSION_GREATER 3.9.6)
    include_guard(GLOBAL)
else()
    if(__CURRENT_FILE_VAR__)
      return()
    endif()
    set(__CURRENT_FILE_VAR__ TRUE)
endif()
```

IMHO, is better to use global variable without checking CMake version but I can add this long construction if you think it's better solution.

Reply:
We can increase minimum cmake version up to 3.13 (we use several features from newer version of cmake, but has `cmake_minimum_required(VERSION 3.7.2 FATAL_ERROR)` which is not correct)

Reply:
ok

Reply:
> zho

yes, just run for ngraph (check and apply format):
make style-check
ngraph/maint/apply-code-format.sh

Reply:
@azhogov I understand, can you fix CI?

Reply:
The fix was merged #1573, so this WA is not needed anymore as well as include_guard in openvino/CMakeLists.txt